packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -0,0 +1,144 @@
+};
+
+export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
+  const searchVersion = useFeature(feature.search);
+  const { sidebarExpanded } = useContext(SettingsContext);
+  const { trackEvent } = useAnalyticsContext();
+  const { completeAction, checkHasCompleted, isActionsFetched } = useActions();
+
+  const isTracked = useRef(false);
+  const shouldShowPulse =
+    isActionsFetched && !checkHasCompleted(ActionType.UsedSearchPanel);
+
+  const [state, setState] = useState(() => {
+    return {
+      provider: SearchProviderEnum.Posts,
+      query: '',
+      isActive: false,
+    };
+  });
+
+  const searchPanelContextValue = useMemo(() => {
+    return {
+      ...state,
+      setProvider: (provider: SearchProviderEnum) => {
+        setState((currentState) => {
+          return { ...currentState, provider };
+        });
+      },
+      setActive: (isActive: boolean) => {
+        setState((currentState) => {
+          return { ...currentState, isActive };
+        });
+      },
+    };
+  }, [state]);
+
+  const showDropdown = state.isActive && state.query.length >= minQueryLength;
+
+  const gradientContainer = useMemo(() => {
+    if (typeof state.isActive === 'undefined') {
+      return undefined;
+    }
+
+    if (typeof document === 'undefined') {
+      return undefined;
+    }
+
+    return (
+      document.getElementById(searchPanelGradientElementId) ||
+      document.createElement('div') // unmounted div to not show the gradient
+    );
+  }, [state.isActive]);
+
+  if (searchVersion !== SearchExperiment.V1) {
+    return null;
+  }
+
+  return (
+    <SearchPanelContext.Provider value={searchPanelContextValue}>
+      <div className={classNames(className, 'relative flex flex-col')}>
+        <SearchPanelInput
+          className={{
+            container: classNames(
+              'w-[35rem]',
+              shouldShowPulse && 'highlight-pulse',
+            ),
+          }}
+          valueChanged={(newValue) => {
+            setState((currentState) => {
+              return { ...currentState, query: newValue };
+            });
+          }}
+          inputProps={{
+            onFocus: () => {
+              searchPanelContextValue.setActive(true);
+
+              if (!isTracked.current && shouldShowPulse) {
+                isTracked.current = true;
+
+                trackEvent({
+                  event_name: AnalyticsEvent.SearchHighlightAnimation,
+                });
+              }
+
+              completeAction(ActionType.UsedSearchPanel);
+            },
+            onBlur: () => {
+              searchPanelContextValue.setActive(false);
+            },
+          }}
+        >
+          <div
+            className={classNames(
+              'absolute top-[3.7rem] w-full items-center rounded-b-2xl border border-theme-divider-quaternary !bg-theme-bg-secondary-blur !bg-opacity-[0.8] px-3 py-2 backdrop-blur-md transition-opacity duration-200 ease-in-out',
```

Comment:
The top not being even annoys me 🤣 
Also rounded-b-2xl should be a numeric one like rounded-b-12 (whatever it is)
As for the background I think we have ones with float that can work.
Let me know if you quickly want to sync, because it's hard to explain (it's something I'll do in the gathering)

Reply:
> The top not being even annoys me 🤣

Me too but its because of the strange position of focused input. I'll take a look if I can adjust it somehow but it is what it is. 

Fixed rounded and as for background yes I'll probs remove it https://github.com/dailydotdev/apps/pull/2510#discussion_r1455465036